simulators/ethereum/engine-gnosis-erigon/main.go

```diff
@@ -13,9 +13,6 @@ import (
 	"github.com/ethereum/hive/simulators/ethereum/engine/helper"
 	"github.com/ethereum/hive/simulators/ethereum/engine/test"
 
-	suite_auth "github.com/ethereum/hive/simulators/ethereum/engine/suites/auth"
-	//suite_engine "github.com/ethereum/hive/simulators/ethereum/engine/suites/engine"
-	suite_ex_cap "github.com/ethereum/hive/simulators/ethereum/engine/suites/exchange_capabilities"
```

Comment:
Why these suites are removed?

Reply:
This changes relates to old `ethereum/engine-gnosis-erigon`, not jq one. It does not work with legacy simulator. We run the suites from jq simulator

Reply:
ah, right. missed the path 😬 

But now I'm not sure why there are changes to two different simulators. I assumed we'd want to use only a `-jq` sim going forward

Reply:
Yes, but some tests like `cancun` still require old simulator. Since there were differences in implementation between nethermind and erigon. I will remove all not-jq simulators when we will be safe (hope next week, when erigon team investigates the failures).